napari/utils/event.py

```diff
@@ -585,8 +585,8 @@ def unblock(self, callback=None):
     def blocker(self, callback=None):
         """Return an EventBlocker to be used in 'with' statements
 
-           Notes
-           -----
+        Notes
+        -----
            For example, one could do::
```

Comment:
@Carreau I'm going to merge the PR but this looks like it missed this indentation, maybe because it was "dependent" on another fix?

Reply:
Or maybe this is the "more conservative with whitespace" version...? Incidentally, I've very much liked all of your whitespace fixes so far...